src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -16,7 +18,9 @@ public ReviewModel(BlobAssemblyRepository assemblyRepository)
 
         public async Task OnGetAsync(string id)
         {
-            AssemblyModel = await assemblyRepository.ReadAssemblyContentAsync(id);
+            string json = await assemblyRepository.ReadAssemblyContentAsync(id);
+            AssemblyAPIV assembly = JsonSerializer.Parse<AssemblyAPIV>(json);
```

Comment:
Move serialization/deserialization code into the repository implementation as it's an implementation detail of how the model is stored.